src/Hpack/Config.hs

```diff
@@ -501,6 +509,20 @@ instance Semigroup Empty where
 instance FromValue Empty where
   fromValue _ = return Empty
 
+data GhcLanguage =
+    Haskell2010
```

Comment:
Why not also allow `Haskell98`?

Reply:
I don't think there is any reason to exclude Haskell98. I only left it out because the existing `hpack` does not accommodate it.

Reply:
I think it should be added.  Otherwise the `verbatim: default-language: Haskell98` hack will prevail.

Reply:
Now added.